jeeves/jobs.py

```diff
@@ -19,7 +41,12 @@ def get_jenkins_job_info(server, job_name, filter_param_name=None, filter_param_
 		job_url = job_info['url']
 		lcb_num = job_info['lastCompletedBuild']['number']
 		tempest_tests_failed = None
+		stage_failure = 'N/A'
 		build_info = server.get_build_info(job_name, lcb_num)
+		if build_info['result'] == 'FAILURE' or build_info['result'] == 'UNSTABLE':
```

Comment:
Shouldn't this only be FAILURE?

Reply:
Yes, removed it and moved also this after filter condition to get stage failures for filtered jobs.